packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -12,6 +12,7 @@ $unselectedHoverColor: $ms-color-neutralLighter;
   display: inline-block;
   min-width: 100%;
```

Comment:
Do we need both `min-width: 100%` and `width: 100%`. If so, it would be good to add a comment explaining why for future reference.